### ISSUE TITLE
Does not start after the update v5.3.1

### ISSUE THREAD

**Issue by bulletproof2k** [theotherp/nzbhydra2#899]
[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/13402816/nzbhydra2.log)
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/13402817/wrapper.log)

**Comment by theotherp** [comment]
Well shit. Thanks, can reproduce.

**Comment by theotherp** [comment]
Seems to affect only windows which is why my build pipeline didn't catch it - it only runs the linux executable.

**Comment by theotherp** [comment]
Please revert to https://github.com/theotherp/nzbhydra2/releases/tag/v5.2.0 for now.

**Comment by bulletproof2k** [comment]
Now the interface does not open from the tray in the same way.
![2023-11-19_13-56-52](https://github.com/theotherp/nzbhydra2/assets/1818579/8fd358cf-803b-4fb7-b812-98dee7a3304f)
[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/13402946/nzbhydra2.log)
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/13402947/wrapper.log)

**Comment by theotherp** [comment]
Try https://github.com/theotherp/nzbhydra2/releases/tag/v5.1.11 then

**Comment by bulletproof2k** [comment]
> Try https://github.com/theotherp/nzbhydra2/releases/tag/v5.1.11 then

It works fine, I'm closing the issue. Please fix all identified issues in the next release. 

Thank you for the work done.

**Comment by theotherp** [comment]
I'm reopening for visibility.

**Comment by theotherp** [comment]
@bulletproof2k Could you please try https://ufile.io/8fhjtme8 that should hopefully work.

**Comment by bulletproof2k** [comment]
Nope

> @bulletproof2k Could you please try https://ufile.io/8fhjtme8 that should hopefully work.

Nope, the same problems as at the beginning of the issue.

**Comment by theotherp** [comment]
I don't understand, I was able to reproduce the original issue but with this it's fixed. 

The folder's content should look like this now:
![image](https://github.com/theotherp/nzbhydra2/assets/14368889/4216c141-4d29-4d9f-ba07-d3b99fafe8cb)

**Comment by bulletproof2k** [comment]
That's how it looked, but I've already rolled back to the previous version.

**Comment by junitodx** [comment]
Also having an issue after the update but my issues is that I can't get the NzbHydra2 service to start. I have tried uninstalling and reinstalling the service and also rolled back to 5.1.11 and 5.1.8 and then uninstalled and reinsalled the service with those CMDs but still have the same issue
![image](https://github.com/theotherp/nzbhydra2/assets/151453565/d3d13c6e-c249-4e08-8251-c89335927277)

**Comment by theotherp** [comment]
Try running the console exe directly from the nzbhydra folder.

**Comment by junitodx** [comment]
> Try running the console exe directly from the nzbhydra folder.

Thanks for the suggestion but still no dice. This happened overnight, everything was working fine when I went to sleep and when i woke up this morning, Sonaar could not connect to NzbHydra and I found the service couldn't be started. If it helps, I am running Windows 7. This is just a PC for downloading and streaming to a Shield so I never updated to Win10 or 11 because I could not possibly remember all the automation and settings/rules I had to set up.
![image](https://github.com/theotherp/nzbhydra2/assets/151453565/091a4333-0e5c-4bbe-9558-6ab6c67872f6)

**Comment by theotherp** [comment]
@JLLeitschuh Please try this one: 
[NZBHydra2 Console.zip](https://github.com/theotherp/nzbhydra2/files/13404378/NZBHydra2.Console.zip)

Windows 7 is actually a corner case and hard to test for me.

**Comment by junitodx** [comment]
> @junitodx Please try this one: [NZBHydra2 Console.zip](https://github.com/theotherp/nzbhydra2/files/13404378/NZBHydra2.Console.zip)
> 
> Windows 7 is actually a corner case and hard to test for me.

So, I no longer get an error when trying to run the NZBHydra2 Console.exe however I still can't get the service to start. I tried to uninstall and reinstall the service and then reboot but it won't start.
![image](https://github.com/theotherp/nzbhydra2/assets/151453565/a9dc94f0-a5d8-4739-81e1-a2061c8f3d15)

**Comment by theotherp** [comment]
Please post wrapper.log and nzbhydra2.log

**Comment by junitodx** [comment]
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/13404410/wrapper.log)
[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/13404411/nzbhydra2.log)

**Comment by theotherp** [comment]
Try `core.exe` from 5.1.1, do not overwrite `nzbhydra2.exe` or `nzbhydra2 console.exe`.

**Comment by theotherp** [comment]
The problem is that you reverted to 5.1.8 and going back is only possible to certain older versions where the config file format hasn't changed too much. You started 5.1.11 which updated the config file and which 5.1.8 is unable to read.

**Comment by bulletproof2k** [comment]
> The problem is that you reverted to 5.1.8 and going back is only possible to certain older versions where the config file format hasn't changed too much. You started 5.1.11 which updated the config file and which 5.1.8 is unable to read.

Your answer refers to which of the participants in the conversation?

**Comment by junitodx** [comment]
> > The problem is that you reverted to 5.1.8 and going back is only possible to certain older versions where the config file format hasn't changed too much. You started 5.1.11 which updated the config file and which 5.1.8 is unable to read.
> 
> Your answer refers to which of the participants in the conversation?

I am guessing me, I definitely reverted back to those versions. Sorry if I hijacked your thread, figured since my issue was also caused after the update I should not have started a new thread. My bad.

**Comment by junitodx** [comment]
> Try `core.exe` from 5.1.11, do not overwrite `nzbhydra2.exe` or `nzbhydra2 console.exe`.

Yep, this fixed me up. Service now stays on and I can access hydra via browser. Appreciate both your guy's help.

**Comment by theotherp** [comment]
Great. I've updated my build scripts to hopefully keep compatibility with Windows 7.

Now if I could explain why the new code still doesn't run for @bulletproof2k ...

**Comment by theotherp** [comment]
Alright, I've tested it on three machines and added windows tests to the build pipeline. I'm gonna release it now. If it still doesn't work for you, @bulletproof2k, we'll have to see what's going on.

**Comment by bulletproof2k** [comment]
> Alright, I've tested it on three machines and added windows tests to the build pipeline. I'm gonna release it now. If it still doesn't work for you, @bulletproof2k, we'll have to see what's going on.

Ok

**Comment by Burken** [comment]
core.exe from 5.1.11 works with the rest from v5.3.2
But after a while: .Starting process to update to v5.3.2
Then:
The current machine does not support all of the following CPU features that are required by the image: [CX8, CMOV, FXSR, MMX, SSE, SSE2, SSE3, SSSE3, SSE4_1, SSE4_2, POPCNT, LZCNT, AVX, AVX2, BMI1, BMI2, FMA].
Please rebuild the executable with an appropriate setting of the -march option.The current machine does not support all of the following CPU features that are required by the image: [CX8, CMOV, FXSR, MMX, SSE, SSE2, SSE3, SSSE3, SSE4_1, SSE4_2, POPCNT, LZCNT, AVX, AVX2, BMI1, BMI2, FMA].
Please rebuild the executable with an appropriate setting of the -march option.

**Comment by bulletproof2k** [comment]
Does not work with overwriting files in the current folder, rolled back 5.1.11.

**Comment by theotherp** [comment]
Please at least post OS, version and so on.

**Comment by bulletproof2k** [comment]
Windows 11 pro

**Comment by theotherp** [comment]
Which CPU?

**Comment by bulletproof2k** [comment]
> Which CPU?

Intel(R) Core(TM) i5-3570 CPU @ 3.40GHz   3.40 GHz

**Comment by theotherp** [comment]
Log please.

**Comment by Burken** [comment]
> Please at least post OS, version and so on.
Intel Xeon E5-2609 v2 with Win Server 2019 17763.5122

**Comment by bulletproof2k** [comment]
[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/13418500/nzbhydra2.log)
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/13418501/wrapper.log)

**Comment by theotherp** [comment]
Got it.
@burken Please try this one: https://ufile.io/6bqwjr25

**Comment by theotherp** [comment]
@bulletproof2k You can try https://ufile.io/6bqwjr25 as well.

**Comment by theotherp** [comment]
Alright. So the problem is that because of an update of a library the program exes for windows have now been compiled to only run on the same architecture on which they have been built which is AMD in my case. I only have AMD machines which is why I didn't catch that. The new exe which I posted above should hopefully work on intel CPUs as well.

I also see a database error in your log, @Burken. So it's possible that the new exe fixes the architecture issue but you may still have problems running hydra. In that case you may have to restore the database file from a backup.

**Comment by Burken** [comment]
> Got it. @Burken Please try this one: https://ufile.io/6bqwjr25

That core.exe is working for me!  -5.3.3-SNAPSHOT

**Comment by theotherp** [comment]
Yeah as it says you need to update the files from https://github.com/theotherp/nzbhydra2/releases/tag/v5.3.2

**Comment by bulletproof2k** [comment]
> Yeah as it says you need to update the files from https://github.com/theotherp/nzbhydra2/releases/tag/v5.3.2

Rewrite the core.exe?

**Comment by theotherp** [comment]
No, just the files mentioned in the popup, the ones starting with `nzbhydra`.

**Comment by theotherp** [comment]
Still haven't found a way to update these files from within the process.

**Comment by junitodx** [comment]
Hmm...looks like I'm down again after the last update. I tried copying only the NZBHydra2.exe and NZBHydra2 Console.exe from here https://github.com/theotherp/nzbhydra2/releases/tag/v5.3.2 but still on good. 
Win7 Ultimate
Intel (R) Core(TM) i3-2120 CPU@3.30GHz
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/13420331/wrapper.log)
[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/13420332/nzbhydra2.log)

**Comment by theotherp** [comment]
@junitodx Make sure to download the current release. Also overwrite `core.exe`.

**Comment by junitodx** [comment]
> @junitodx Make sure to download the current release. Also overwrite `core.exe`.

Thanks that worked. Since bulletproof2k wasn't supposed to overwrite the core.exe I thought I only needed to overwrite the NZBHydra2.exe and NZBHydra2 Console.exe files

**Comment by bulletproof2k** [comment]
@[theotherp](https://github.com/theotherp) should I overwrite all files too?

**Comment by theotherp** [comment]
Sure, go for it. The exe files are important.

**Comment by quidnun** [comment]
I'm having the same problem on AMD with the service pausing?
